docs/quickstart.rst

```diff
@@ -57,15 +57,20 @@ deprecation indicators are now available.
     **Shortcut!**
 
     In order to bypass the installation and configuration of *Redis* required by the
-    default storage and cache, use the «in-memory» backend in :file:`myproject.ini`:
+    default storage and cache, use the «in-memory» backend in :file:`development.ini`:
 
     .. code-block:: ini
 
-        # myproject.ini
+        # development.ini
         cliquet.cache_backend = cliquet.cache.memory
         cliquet.storage_backend = cliquet.storage.memory
 
 
+Now is a good time to install the project::
+
+    $ python setup.py develop
```

Comment:
+1 for using pip everywhere rather than directly the setup.py call.

For completeness, [apparently](https://pip.pypa.io/en/stable/reference/pip_install.html#editable-installs) the difference between `pip install -e` and `python setup.py develop` is really tiny, and it will act exactly the same in this case: 

> The "SomeProject.egg-info" directory is created relative to the project path. This is one advantage over just using setup.py develop, which creates the "egg-info" directly relative the current working directory.